tests/unittest/test_language_tag_factory.py

```
@@ -0,0 +1,47 @@
+    language_tag_factory: LanguageTagFactory | None
+
+    def setUp(self) -> None:
+        self.language_tag_factory = LanguageTagFactory()
+
+    def tearDown(self) -> None:
+        self.language_tag_factory = None
+
+    def test_get(self) -> None:
+        language_tag: LanguageTag = self.language_tag_factory.get_tag(tag="zh-Hant")
+
+        self.assertEqual(language_tag.get_bcp_47_tag(), "zh-Hant")
```

Ruff (0.11.9)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)